packages/syft/src/syft/service/settings/settings.py

```diff
@@ -265,6 +270,30 @@ class ServerSettings(SyftObject):
     pwd_token_config: PwdTokenResetConfig = PwdTokenResetConfig()
     allow_guest_sessions: bool = True
 
+    @field_validator("admin_email")
+    def admin_email_format(cls, v: str) -> str:
+        try:
+            validate_email(v)
+        except EmailNotValidError as e:
+            raise ValueError(f"Invalid email format: {e}")
+        return v
```

Comment:
Shouldn't this be achieved with just

```py
admin_email: EmailStr
```
?

Reply:
Actually that was my first approach but it caused other issues like unclear error messages when updating the settings due to autosplat, or the validation not being triggered because of how `ServerSettingsUpdate` data is transfered to the `ServerSettings` object, so I went back to `str`

The `field_validator` was the first alternative I tried (which didn't fix the side effects either). 

Eventually, the fixes I implemented for the other issues allowed the validator to do it's job but I didn't revisit it to check if I could replace it with a simpler `EmailStr`.

However, prompted by your comment I realized that now I can, so I will. Thanks!